CHANGES.md

```diff
@@ -1,3 +1,9 @@
+### unreleased
+
+#### Changes
+
+  + Revert depreciation of the `doc-comments` option (#1331) (Jules Aguillon)
```

Comment:
I think it should be "deprecation", can you also link #1293 so we can easily jump back and forth? thanks

Reply:
:+1: